duties.md

```diff
@@ -0,0 +1,24 @@
+# Team duties
+
+This document is helpful to know what areas are covered by each team member.
+
+## [Mostafa](https://github.com/mostafa-anm)
+
+- first-level tech support (via Google Groups)
+- tech meetups organization
+- tech interviews
+- GA
```

Comment:
This is to board, y'all responsible to track your own features. However, Mos is a knowledge bearer. Maybe we split those. That being said, Bianka will be responsible actually monitoring the data in GA4. We are probably more responsible for GTM.